[2023-08-26T04:05:56.615+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: nikeProducts_dag.extract_task manual__2023-08-26T03:46:46.645737+00:00 [queued]>
[2023-08-26T04:05:56.624+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: nikeProducts_dag.extract_task manual__2023-08-26T03:46:46.645737+00:00 [queued]>
[2023-08-26T04:05:56.625+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-08-26T04:05:56.626+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2023-08-26T04:05:56.626+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-08-26T04:05:56.639+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): extract_task> on 2023-08-26 03:46:46.645737+00:00
[2023-08-26T04:05:56.649+0000] {standard_task_runner.py:55} INFO - Started process 753 to run task
[2023-08-26T04:05:56.656+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'nikeProducts_dag', 'extract_task', 'manual__2023-08-26T03:46:46.645737+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/nikeProducts_dag.py', '--cfg-path', '/tmp/tmp78gpnoos']
[2023-08-26T04:05:56.661+0000] {standard_task_runner.py:83} INFO - Job 116: Subtask extract_task
[2023-08-26T04:05:56.744+0000] {task_command.py:376} INFO - Running <TaskInstance: nikeProducts_dag.extract_task manual__2023-08-26T03:46:46.645737+00:00 [running]> on host 9f01d644bb11
[2023-08-26T04:05:56.815+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nikeProducts_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2023-08-26T03:46:46.645737+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-26T03:46:46.645737+00:00
[2023-08-26T04:05:56.816+0000] {logging_mixin.py:120} INFO - nike_sales_2023_08_22.csv mapped to products_etl.csv
[2023-08-26T04:05:56.819+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/products_scripts/get_data.py", line 17, in main
    df = pd.read_csv(os.path.join(AIRFLOW_HOME, "dags", "files", "input", "products", input_file))
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dags/files/input/products/nike_sales_2023_08_22.csv'
[2023-08-26T04:05:56.833+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=nikeProducts_dag, task_id=extract_task, execution_date=20230826T034646, start_date=20230826T040556, end_date=20230826T040556
[2023-08-26T04:05:56.846+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 116 for task extract_task ([Errno 2] No such file or directory: '/opt/airflow/dags/files/input/products/nike_sales_2023_08_22.csv'; 753)
[2023-08-26T04:05:56.867+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-08-26T04:05:56.904+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
